styles/layout/_flex.scss

```diff
@@ -0,0 +1,8 @@
+/**
+ * Flex
+ * =====
+ */
+
+.align-items-center {
```

Comment:
why

Reply:
Why what?

Reply:
Alright, but it can be set along with the flex related stuff as a `center` class